### PR TITLE
Add paragraph about replay protection

### PR DESCRIPTION
**Pull request by dthaler**
Also fix references to TEEP architecture RFC

### CODE REVIEW

File: draft-ietf-teep-protocol.md [hunk 1]
```diff
@@ -1779,6 +1779,23 @@ TEEP Broker
...
+  The transport protocol under the TEEP protocol might provide replay
+  protection, but may be terminated in the TEEP Broker which is not trusted
+  by the TEEP and so the TEEP protocol does replay protection itself.
+  If attestation of the TAM is used, the attestation freshness mechanism
+  provides replay protection for attested QueryRequest messages.
+  If non-attested QueryRequest messages are replayed, the TEEP Agent will generate
+  QueryResponse or Error messages, but the REE can already conduct Denial of Service
+  attacks against the TEE and/or the TAM even without the TEEP protocol.
+  QueryResponse messages have replay protection via attestation freshness mechanism or (if
+  attestation is not used) the token field in the message.
```
Comment (mingpeiwk) [inline draft-ietf-teep-protocol.md:1793]: How about reorder it as follows?

QueryResponse messages have replay protection via attestation freshness mechanism or the token field in the message if attestation is not used.

**Review by mingpeiwk — APPROVED** [review]
Minor comment on describing replay Request protection explicitly for non-attested case.

File: draft-ietf-teep-protocol.md (hunk 1 above)
Comment (dthaler) [reply draft-ietf-teep-protocol.md]: Updated

File: draft-ietf-teep-protocol.md [hunk 2]
```diff
@@ -1779,6 +1779,23 @@ TEEP Broker
   Agents are protected against such downgrade attacks based on
   features offered by the manifest itself.
 
+Replay Protection
+: TEEP supports replay protection as follows.
```
Comment (hannestschofenig) [inline draft-ietf-teep-protocol.md:1783]:
```suggestion
: The TEEP protocol supports replay protection as follows.
```

File: draft-ietf-teep-protocol.md [hunk 3]
```diff
@@ -1779,6 +1779,23 @@ TEEP Broker
...
+  The transport protocol under the TEEP protocol might provide replay
+  protection, but may be terminated in the TEEP Broker which is not trusted
+  by the TEEP and so the TEEP protocol does replay protection itself.
```
Comment (hannestschofenig) [inline draft-ietf-teep-protocol.md:1786]:
```suggestion
  by the TEEP architecture and so the TEEP protocol does replay protection itself.
```

**Review by hannestschofenig — APPROVED** [review]
Thanks for writing this PR. I just made a few minor text proposals.

File: draft-ietf-teep-protocol.md [hunk 4]
```diff
@@ -1779,6 +1779,23 @@ TEEP Broker
   Agents are protected against such downgrade attacks based on
   features offered by the manifest itself.
 
+Replay Protection
+: The TEEP protocol supports replay protection as follows.
+  The transport protocol under the TEEP protocol might provide replay
+  protection, but may be terminated in the TEEP Broker which is not trusted
+  by the TEEP and so the TEEP protocol does replay protection itself.
```
Comment (dthaler) [inline draft-ietf-teep-protocol.md:1786]:
```suggestion
  by the TEEP Agent and so the TEEP protocol does replay protection itself.
```

File: draft-ietf-teep-protocol.md (hunk 3 above)
Comment (dthaler) [reply draft-ietf-teep-protocol.md:1786]: actually should be "TEEP Agent" not "TEEP architecture"

**Pull request merged**